### PR TITLE
docs: Hint on Windows as development OS

### PR DESCRIPTION
**Pull request by Neudrino**
## Summary

In context of https://github.com/magma/magma/issues/10116 this is a small attempt to make the difficulties using Windows as development system known and avoid trouble for new joiners.

* Added Link from prerequisits to developer onboarding page.
* Added comment on the 3 major PC platforms: Mac OS, Linux OS, Windows OS.

## Additional Information

This is the first attempt to document some of the experience from investigations on Windows OS as development platform.

I hope it does not collide with https://github.com/magma/magma/issues/9848 and overall changes to be done there.

### CODE REVIEW

**Comment by Neudrino** [comment]
Cannot add @ildikov as reviewer. Instead tagging here, as it might be of interest.

**Review by electronjoe — APPROVED** [review]

**Comment by electronjoe** [comment]
Thank you for the documentation assistance @Neudrino

**Review by ildikov — APPROVED** [review]

File: docs/readmes/basics/prerequisites.md [hunk 1]
```diff
@@ -6,7 +6,13 @@ hide_title: true
 # Prerequisites
 
 These are the prerequisites to setting up a full private LTE Magma deployment.
-Additional prerequisites for developers can be found in the developer's guide.
+Additional prerequisites for developers can be found in the [developer's guide](../contributing/contribute_onboarding.md).
+
+## Operating System
+
+Currently, the main development operating system (OS) is **Mac OS**. Documentation is mainly focused on that operating system.
```
Comment (hcgatewood) [inline docs/readmes/basics/prerequisites.md]:
```suggestion
Currently, the main development operating system (OS) is **macOS**. Documentation is mainly focused on that operating system.
```

File: docs/readmes/basics/prerequisites.md [hunk 2]
```diff
@@ -6,7 +6,13 @@ hide_title: true
...
+To develop on a **Linux OS**, the package manager (brew for Mac OS) will need to be replaced by the appropriate package manager for the respective Linux distribution (e.g. apt, yum, etc.) where necessary.
```
Comment (hcgatewood) [inline docs/readmes/basics/prerequisites.md]:
```suggestion
To develop on a **Linux OS**, the package manager (brew for macOS) will need to be replaced by the appropriate package manager for the respective Linux distribution (e.g. apt, yum, etc.).
```

File: docs/readmes/basics/prerequisites.md [hunk 3]
```diff
@@ -6,7 +6,13 @@ hide_title: true
...
+**Windows OS** is currently _not_ supported as developing environment, due to some dependencies on Linux only tools during setup, such as Ansible or `fcntl`. You can try to use a [DevContainer setup](../contributing/contribute_vscode.md#open-a-devcontainer-workspace-with-github-codespaces) though.
```
Comment (hcgatewood) [inline docs/readmes/basics/prerequisites.md]:
```suggestion
**Windows OS** is currently _not_ supported as developing environment, due to some dependencies on Linux-only tools during setup, such as Ansible or `fcntl`. You can try to use a [DevContainer setup](../contributing/contribute_vscode.md#open-a-devcontainer-workspace-with-github-codespaces) though.
```

**Review by hcgatewood — APPROVED** [review]
Awesome thanks!

**Comment by Neudrino** [comment]
DCO check was failing with the inline changes. So I did force push to get the CI green. My appologies, @themarwhal . 😇 

And thanks @hcgatewood for your detailed review! 😃

**Pull request merged**